insights/parsers/lspci.py

```diff
@@ -69,14 +70,20 @@ def parse_content(self, content):
         # Parse kernel driver lines
         self.data = {}
         bus_device_function = ""
+        bus_device_function_re = re.compile(r'[0-9a-f]+:[0-9a-f]+.[0-9a-f]+')
+
+        fields = ["Subsystem", "Kernel driver in use", "Kernel modules"]
+
         for line in get_active_lines(content):
             parts = line.split(None)
-            if parts[0] and parts[0] not in ['Subsystem:', 'Kernel'] \
-                    and len(parts[0].split(':')) == 2 and len(parts[0].split('.')) == 2:
+            if not parts:
+                continue
+
+            if parts[0] and bus_device_function_re.match(parts[0]):
```

Comment:
@akshay196 its not necessary to check `parts[0]` here since its already been checked and confirmed its not empty.

Reply:
Done